apps/crn-frontend/src/network/teams/TeamOutput.tsx

```diff
@@ -57,7 +57,11 @@ const TeamOutput: React.FC<TeamOutputProps> = ({
 
   const createResearchOutput = usePostResearchOutput({ publish: true });
   const createDraftResearchOutput = usePostResearchOutput({ publish: false });
-  const updateResearchOutput = usePutResearchOutput();
+
+  const updateResearchOutput = usePutResearchOutput({
+    publish: !researchOutputData?.published,
```

Comment:
This doesn't seem like it will work. Won't edits of unpublised RO make the RO unpublished?